compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -4374,6 +4377,64 @@ private STNode parseObjectTypeDescriptor() {
                 objectMembers, closeBrace);
     }
 
+    /**
+     * Parse object constructor expression.
+     *
+     * object-constructor-expr :=
+     *    [annots] [client] object [type-reference] {
+     *       object-member*
+     *    }
+     * object-member := object-field | method-defn
+     *
+     * @param isRhsExpr
+     * @param annots
+     * @return Parsed node
+     */
+    private STNode parseObjectConstructorExpressionNode(boolean isRhsExpr, STNode annots) {
```

Comment:
Shall we change method `parseObjectConstructorExpressionNode` to `parseObjectConstructorExpression` to be aligned with other expressions.